src/util/loanSearch/filters/distributionModels.js

```diff
@@ -4,21 +4,24 @@ import { getEnumNameFromQueryParam } from '@/util/loanSearch/queryParseUtils';
 // Distribution model enum keys
 export const FIELDPARTNER_KEY = 'FIELDPARTNER';
 export const DIRECT_KEY = 'DIRECT';
+export const BOTH = 'BOTH';
```

Comment:
Interesting that this is needed, since I did test saving with this filter as null. Was there an error?

Reply:
In my testing saved search urls were coming back with field partner if null was used. Direct worked but "All" was getting converted to field partner, likely due to the backend handling. There's some monolith code updates to go with this, which checks for `both` as the value before forcing a default of field partner if null is present...

Reply:
Ah gotcha. I had assumed that was the auto-application of partner that the data team mentioned in slack recently, but makes sense that this was actually from the `null`.